docs/plugins/composable_resources.md

```diff
@@ -276,6 +271,31 @@ acPostProcForDataCopyReceived(*leaf_resource) {
 }
 ```
 
+#### Detached Mode Configuration
+
+Detached mode was added as a configuration option for UnixFileSystem resources in release 4.3.1.  When detached mode is enabled, any server may serve up requests for the resource as long as all servers share a common mounted vault.
+
+To enable detached mode, the **host_mode** setting in the context string should be set to **detached** as in the following example:
```

Comment:
What happens if we set it to "attached" or "banana" or empty string? Worth noting?

Reply:
Anything other than "detached" will default to attach mode.

Reply:
Okay. I feel that this may be worth a mention in the actual docs.

Reply:
agreed, just a sentence that says...

If "host_mode" is not set to "detached", the unixfilesystem plugin will default to "attached" mode.

Reply:
I added the sentence in.  Please take a look at where I added it and the use of quotes.  (I didn't include quotes around "host_mode" as we didn't do it elsewhere nor highlighted it because it has already been highlighted.  I also didn't use quotes around "attached" because we haven't stated that it is a setting with meaning.)